susemanager/src/mgr-create-bootstrap-repo

```diff
@@ -765,6 +765,16 @@ def main():
         if opts.create not in mgr_bootstrap_data.DATA:
             log_error("'%s' not found" % opts.create)
             sys.exit(1)
+        if opts.flush:
+            destdir = os.path.normpath(mgr_bootstrap_data.DATA[opts.create]['DEST'])
+            if os.path.exists(destdir):
+                dirprefix, lastdir = os.path.split(destdir)
+                destdirold = os.path.join(dirprefix, "{0}.{1}".format(lastdir, "old"))
+                if opts.dryrun:
+                    log("Move directory: {0}".format(destdir))
+                else:
+                    log("FLUSH: move destdir '{0}' to old".format(destdir))
+                    os.rename(destdir, destdirold)
```

Comment:
Just a bit nitpick, maybe use `os.replace` instead here?

Reply:
I know rename, but how is replace different?

Reply:
I will keep rename for now as we plan a bigger refactoring of this anyway later.